content/en/docs/contribute/localization.md

```diff
@@ -54,16 +54,16 @@ Once you've opened a localization PR, you can become members of the Kubernetes G
 
 ### Add your localization team in GitHub
 
-Next, add your Kubernetes localization team to [`sig-docs/teams.yaml`](https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-docs/teams.yaml). For an example of adding a localization team, see the PR to add the [Spanish localization team](https://github.com/kubernetes/org/pull/685). 
+Next, add your Kubernetes localization team to [`sig-docs/teams.yaml`](https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-docs/teams.yaml). For an example of adding a localization team, see the PR to add the [Spanish localization team](https://github.com/kubernetes/org/pull/685).
 
-Members of `sig-docs-**-owners` can approve PRs that change content within (and only within) your localization directory: `/content/**/`. 
+Members of `@kubernetes/sig-docs-**-owners` can approve PRs that change content within (and only within) your localization directory: `/content/**/`.
 
-The `sig-docs-**-reviews` team automates review assignment for new PRs.
+For each localization, The `@kubernetes/sig-docs-**-reviews` team automates review assignment for new PRs.
```

Comment:
nit: line 61, lowercase, the
Why is @kubernetes needed as a prefix to sig-docs

Reply:
> Why is @kubernetes needed as a prefix to sig-docs?
That's how you'd type it into a GitHub comment to mention the team. I think that's a useful way to write it.